highlighters/main/test-data/alias-comment1.zsh

```diff
@@ -0,0 +1,37 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2016 zsh-syntax-highlighting contributors
```

Comment:
2018 (et seq).

Reply:
No. Copyright year doesn't get incremented unless a copyrightable change is made. Rebase doesn't count.

Reply:
main highlighter does need a bump.

Reply:
Done

Reply:
Ah, didn't realize this branch has been around that long.